src/Mod/Part/App/TopoShape.cpp

```diff
@@ -1661,6 +1661,7 @@ bool TopoShape::analyze(bool runBopCheck, std::ostream& str) const
 #endif
 #if OCC_VERSION_HEX >= 0x060900
             BOPCheck.SetParallelMode(true); //this doesn't help for speed right now(occt 6.9.1).

```

Comment:
@mwganson is this comment still relavent?

Reply:
I think not.